### DISCUSSION TITLE
Cleanup of ARGOCD APPs not working

### DISCUSSION THREAD

**Discussion by AnubhavSabharwa** [argoproj/argo-cd#17570] [General]
Hello ALL,

I am following up App of Apps strategy.

We are creating root yaml and saying sync from here which is nothing but a git path in which all manifest we are adding.

Now, if someone is directly deleted the application Manifest then the apps is getting removed from Argocd which is expected however, the resources are still running.

Is there any possibility to delete the resources if someone deletes the application manifest from git

**Comment by rumstead** [comment]
Take a look at a cascading delete via the finalizer. 

https://argo-cd.readthedocs.io/en/stable/user-guide/app_deletion/#about-the-deletion-finalizer
